h5p-bildetema-words-grid-view/package.json

```diff
@@ -27,10 +23,7 @@
     "typecheck": "tsc"
   },
   "dependencies": {
-    "h5p-types": "^1.18.1",
-    "h5p-utils": "^3.2.1",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
```

Comment:
All projects use these, thus they are "common". I decided to move them into there. They will be available still, as they will be placed in the root node_modules.